[2023-03-13T01:34:56.384+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: validacao_pu_v2.task_calcular_pu_externo scheduled__2023-03-10T00:00:00+00:00 [queued]>
[2023-03-13T01:34:56.398+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: validacao_pu_v2.task_calcular_pu_externo scheduled__2023-03-10T00:00:00+00:00 [queued]>
[2023-03-13T01:34:56.398+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-03-13T01:34:56.398+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 6
[2023-03-13T01:34:56.398+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-03-13T01:34:56.418+0000] {taskinstance.py:1300} INFO - Executing <Task(_PythonDecoratedOperator): task_calcular_pu_externo> on 2023-03-10 00:00:00+00:00
[2023-03-13T01:34:56.425+0000] {standard_task_runner.py:55} INFO - Started process 2203 to run task
[2023-03-13T01:34:56.428+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'validacao_pu_v2', 'task_calcular_pu_externo', 'scheduled__2023-03-10T00:00:00+00:00', '--job-id', '356', '--raw', '--subdir', 'DAGS_FOLDER/examples/validacao_pu_v2.py', '--cfg-path', '/tmp/tmpocbkelcv']
[2023-03-13T01:34:56.429+0000] {standard_task_runner.py:83} INFO - Job 356: Subtask task_calcular_pu_externo
[2023-03-13T01:34:56.486+0000] {task_command.py:388} INFO - Running <TaskInstance: validacao_pu_v2.task_calcular_pu_externo scheduled__2023-03-10T00:00:00+00:00 [running]> on host 40565fc59ef6
[2023-03-13T01:34:56.559+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=alexandre.yoshimatsu
AIRFLOW_CTX_DAG_ID=validacao_pu_v2
AIRFLOW_CTX_TASK_ID=task_calcular_pu_externo
AIRFLOW_CTX_EXECUTION_DATE=2023-03-10T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-03-10T00:00:00+00:00
[2023-03-13T01:34:56.568+0000] {logging_mixin.py:137} INFO - ['xyz', 'abc', 'dfg', 'hij', 'opq']
[2023-03-13T01:34:56.568+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-03-13T01:34:56.575+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=validacao_pu_v2, task_id=task_calcular_pu_externo, execution_date=20230310T000000, start_date=20230313T013456, end_date=20230313T013456
[2023-03-13T01:34:56.600+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2023-03-13T01:34:56.624+0000] {taskinstance.py:2578} INFO - 1 downstream tasks scheduled from follow-on schedule check
